tfe/resource_tfe_sentinel_policy.go

```diff
@@ -125,22 +136,29 @@ func resourceTFESentinelPolicyRead(d *schema.ResourceData, meta interface{}) err
 func resourceTFESentinelPolicyUpdate(d *schema.ResourceData, meta interface{}) error {
 	tfeClient := meta.(*tfe.Client)
 
-	if d.HasChange("enforce_mode") {
+	changeEnforce := d.HasChange("enforce_mode")
+	if d.HasChange("description") || changeEnforce {
 		// Create a new options struct.
-		options := tfe.PolicyUpdateOptions{
-			Enforce: []*tfe.EnforcementOptions{
+		options := tfe.PolicyUpdateOptions{}
+
+		if changeEnforce {
```

Comment:
I'm checking this again inside of the existing conditional because TFE will always create a new policy config when it receives an `enforce` param. So although we want to submit this API request if either the description or enforce changed, I don't want to set `enforce` unless it has changed, because I don't want TFE to be needlessly creating additional policy configurations

Reply:
Yeah I understand 👍